repo/maintenancestats/stats_cleanup_markers.go

```diff
@@ -0,0 +1,33 @@
+package maintenancestats
+
+import (
+	"fmt"
+
+	"github.com/kopia/kopia/internal/contentlog"
+)
+
+const cleanupMarkersStatsKind = "cleanupMarkersStats"
+
+// CleanupMarkersStats are the stats for cleaning up markers.
+type CleanupMarkersStats struct {
+	DeletedEpochMarkerBlobCount       int `json:"deletedEpochMarkerBlobCount"`
+	DeletedDeletionWaterMarkBlobCount int `json:"deletedDeletionWaterMarkBlobCount"`
```

Comment:
From a prior comment, this can be shortened:

```suggestion
	DeletedWaterMarkBlobCount int `json:"deletedWaterMarkBlobCount"`
```

Reply:
Let's merge this PR as is. A followup PR will address the naming of these fields.

Reply:
- field naming addressed in #4911